secrets/secrets.go

```diff
@@ -28,6 +32,9 @@ const currentSecretGeneration = "current-secrets-generation"
 const previousSecretName = "previous-secrets-name"
 const configVersion = "config-version"
 
+// The generatorInputSuffix is appended to a secret name to store the generator config used to create the current value
+const generatorInputSuffix = ".-"
```

Comment:
Forgot to mention: Could we get a more human-readable suffix (containing magic is still fine), so people can tell what it's for?

Reply:
Sure, suggestions welcome. There is no magic here; I just picked something that should not appear in our normal config variable names. I guess anything starting with a dot should be fine. So `.generator`?